.github/workflows/docs.yml

```diff
@@ -40,14 +37,14 @@ jobs:
         with:
           # Need to fetch more than the last commit so that versioneer can
           # create the correct version string. If the number of commits since
-          # the last release is greater than this, the version will still be wrong.
+          # the last release is greater than this, the version still be wrong.
```

Comment:
"the version still be wrong" sounds weird, shouldn't the _to be_ verb be conjugated?

Reply:
On `pypi.yml` the sentence is _"the version will still be wrong"_.

Reply:
Yeah, this was a git merge error 🙁